packages/server/README.md

```diff
@@ -1 +1,51 @@
 # @crossmint/server-sdk
+
+This SDK provides a set of tools for authenticating users in a Crossmint-powered application using server-side rendering (SSR). It simplifies the process of handling authentication tokens and managing user sessions, making it easier to integrate authentication into your Next.js applications.
+
+## Installation
+
+To install the SDK, you can use npm or yarn:
+
+```bash
+npm install @crossmint/server-sdk
+```
+
+## Usage
+
+To use the SDK in your application, follow these steps:
+
+1. Import the SDK into your project:
+
+```ts
+import { createCrossmint, CrossmintAuth } from "@crossmint/server-sdk";
+
+const crossmint = createCrossmint({
+    apiKey: process.env.SERVER_CROSSMINT_API_KEY || "",
+});
+
+const crossmintAuth = CrossmintAuth.from(crossmint);
```

Comment:
maybe this is in the doc, but is there a reason to prefer `CrossmintAuth.from(crossmint)` over `new CrossmintAuth(crossmint)`? Is this a secondary instantiation route?